sal_interfaces/ATCamera/ATCamera_Commands.xml

```diff
@@ -153,4 +153,48 @@
     <Subsystem>ATCamera</Subsystem>
     <EFDB_Topic>ATCamera_command_stop</EFDB_Topic>
   </SALCommand>
+  <!-- Playlist related commands, only valid when using emulated DAQ -->
+  <SALCommand>
+    <Subsystem>ATCamera</Subsystem>
+    <EFDB_Topic>ATCamera_command_play</EFDB_Topic>
+    <item>
+      <EFDB_Name>playlist</EFDB_Name>
+      <Description>Name of an existing DAQ playlist to play</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>repeat</EFDB_Name>
+      <Description>True if the playlist should run in repeat mode</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+  </SALCommand>
+  <SALCommand>
+    <Subsystem>ATCamera</Subsystem>
+    <EFDB_Topic>ATCamera_command_playlist</EFDB_Topic>
+    <item>
+      <EFDB_Name>playlist</EFDB_Name>
+      <Description>Name of a DAQ playlist to be defined (replaced if exists)</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>folder</EFDB_Name>
+      <Description>Folder where images exist in DAQ 2-day store (typically emu)</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>images</EFDB_Name>
+      <Description>: delimited list of images to add to the playlist</Description>
```

Comment:
image names for playlists are the names in the DAQ 2-day store, but for images which were triggered by CCS (or indirectly by OCS) the image name in the 2-day store will normally be the image name as used by CCS/OCS.  For images which came from (e.g. phosim) they could be anything. Note sure I can easily summarize this in a comment right now, I think it will become clearer once we have some experience of using it.